verified-broadcast/.env

```diff
@@ -1,18 +1,28 @@
-# Variables for function ".env"
-# ---
-# description: SID of your Twilio Notify Service
-# format: sid
-# link: https://www.twilio.com/console/notify/services
+# description: The Twilio phone number to send broadcast SMS from
+# format: phone_number
 # required: true
-BROADCAST_NOTIFY_SERVICE_SID=
+TWILIO_PHONE_NUMBER=
```

Comment:
We need a phone number if we want to auto generate all the resources because CodeExchange doesn't support picking a Messaging Service yet but it does allow phone numbers. So it's a proxy. If the phone number you pick already exists inside a Messaging Service we'll use this one, otherwise we'll create one and add that phone number